cli/utilities/packages.py

```diff
@@ -83,6 +86,25 @@ def install(self, pkg, nogpgcheck=False, env_vars={}):
         elif out:
             raise PackageError(f"Failed to install package '{pkg}'")
 
+    def remove(self, pkgs, nogpgcheck=False):
+        """Remove a package or packages
+
+        Args:
+            pkg (list): package needs to be removed
+            env_vars (dict): dictiory with environment variables
+        """
+        pkgs = pkgs if type(pkgs) in (list, tuple) else [pkgs]
+
+        for pkg in pkgs:
```

Comment:
For loop not required. Use below command -
```
cmd += f"{self.manager} remove -y {pkg.join(' ')}"
```

Reply:
Addressed the comment.